src/encoded/auth0.py

```diff
@@ -96,6 +93,40 @@ def forget(self, request):
         return []
 
 
+def signup(request):
+    """
+    Create new user.
+
+    :param request: Pyramid request object
+    """
+    domain = 'encode.auth0.com'
+    url = "https://{domain}/userinfo?access_token={access_token}".format(domain=domain, access_token=request.json['accessToken'])
+    user_data_request = requests.get(url)
+
+    if user_data_request.status_code != 200:
+        raise HTTPBadRequest(explanation='Could not get user data')
+
+    users = request.registry[COLLECTIONS]['user']
```

Comment:
Maybe place this closer to create_item if it's only used for `users.type_info`?

Reply:
I will look into it.

Reply:
Done.